src/api/twitterApi.ts

```diff
@@ -0,0 +1,104 @@
+import { Client } from 'twitter-api-sdk'
+
+interface TwitterUser {
+  id: string
+  name: string
+  followersCount: number
+}
+
+interface TwitterPost {
+  id: string
+  retweetCount: number
+  likeCount: number
+}
+
+const twitterClient = new Client('auth-token')
+
+class TwitterApiService {
```

Comment:
we don't need this class. You can omit it, put the SORT_BY_XY as const variables and also omit the the public and static access modifiers.